_members/02-ryan-corbett.md

```diff
@@ -1,11 +1,19 @@
 ---
 name: Ryan Corbett, Ph.D.
-image: images/photo.jpg
+image: images/corbett_photo.jpg
 description: Senior Bioinformatics Scientist
 role: programmer
 links:
   email: 
   twitter: 
```

Comment:
You can remove this since no profile. While you are welcome to add your email, we can also add GitHub or LinkedIn as other links - whatever you want to display! The types of things you can display are in this yaml file: https://github.com/rokitalab/rokitalab.com/blob/main/_data/types.yaml
```suggestion
```

I actually just looked back and my page has a nonlink to the home page (oops), and just my ORCID, but also because my contact info is at the bottom of the site.

Reply:
ok, I've just added my GitHub username for now